features-and-enhancements-4.5 (#193)
AxonIQ/reference-guide: axon-framework/tuning/event-snapshots.md

> For both non-Spring and Spring users a default `Snapshotter` is provided.
> The former uses the Configuration API to provide a default `AggregateSnapshotter`, retrieving the aggregate factories from the registered Aggregates / `AggregateConfiguration`s.
> Spring uses a `SpringAggregateSnapshotter`, which will automatically looks up the right `AggregateFactory` instances from the application context when a snapshot needs to be created.
>

> > The @Revision is a dedicated `SnapshotFilter` implementation that checks for any revisions in the snapshot. This is used to filter out non-matching snapshotters from the `Repository`'s loading process. 
> > When the @Revision is used on an aggregate the snapshots will be filtered out automatically. 

Reviewer: ```suggestion
> The `@Revision` annotation has a dedicated, automatically configured `SnapshotFilter` implementation. This implementation is used to filter out non-matching snapshots from the `Repository`'s loading process. 
> So when the `@Revision` annotation is used on an aggregate the snapshots will be filtered out automatically. 
```
